README.md

```diff
@@ -0,0 +1,5 @@
+# TensorFlow benchmarks
+This repository contains various TensorFlow benchmarks. Currently, it consists of two projects:
+
+1. [scripts/keras_benchmarks](https://github.com/tensorflow/benchmarks/tree/master/scripts/keras_benchmarks): The Keras benchmarks contain benchmarks for several models using Keras.
```

Comment:
These are not maintained and run against the original Keras project not tf.keras, which is much faster and will be more strongly supported going forward.  I would consider not having this first and maybe even put it under a header as soon to be deprecated tests.

Reply:
OK, moved it second and added a sentence that it is unmaintained and deprecated. What do you think now?

Reply:
Swap keras to the second position so users do not have to read about it if they are looking for tf_cnn_benchmarks.  I realize that is not alphabetical, but it is pragmatic.  :-)

Reply:
Already did. Look at the latest commit in the PR.